pkg/api/meta/help.go

```diff
@@ -39,6 +39,14 @@ func GetItemsPtr(list runtime.Object) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// if we're a runtime.Unstructured, check to see if we have an `items` key
+	if unstructured, ok := list.(*runtime.Unstructured); ok {
```

Comment:
@smarterclayton `api/meta` changes to more cleanly handle the way the unstructured decoder works.  It doesn't make an `UnstructuredList` by default, which is annoying.